maintainers/all-owners.sh

```diff
@@ -66,15 +66,55 @@ all_owners_raw()
     done
 }
 
+# Find e-mail address of the commiter based on theire name and the DCO
```

Comment:
```suggestion
# Find e-mail address of the committer based on their name and the DCO
```

Reply:
Also, "the" in "the DCO" should be dropped.

Reply:
Done and done.